test/e2e/specs/onboarding/signup__difm.ts

```diff
@@ -0,0 +1,88 @@
+/**
+ * @group calypso-pr
+ */
+
+import { DataHelper, DIFMFlow, CartCheckoutPage, TestAccount } from '@automattic/calypso-e2e';
+import { Browser, Page } from 'playwright';
+
+declare const browser: Browser;
+
+describe( DataHelper.createSuiteTitle( 'Do it for me' ), () => {
+	let page: Page;
+	let difmFlow: DIFMFlow;
+	let cart: CartCheckoutPage;
+
+	beforeAll( async () => {
+		page = await browser.newPage();
+		difmFlow = new DIFMFlow( page );
+		cart = new CartCheckoutPage( page );
+
+		const testAccount = new TestAccount( 'defaultUser' );
+		await testAccount.authenticate( page );
+	} );
+
+	/**
+	 * Navigate to initial setup page.
+	 */
+	const navigateToLanding = () => {
+		it( `Navigate to landing page`, async () => {
+			await difmFlow.visitSetup();
+			await difmFlow.validateStartPage();
+		} );
+	};
+
+	describe( 'Follow the default DIFM order flow', () => {
+		navigateToLanding();
+
+		it( 'Start a DIFM order for a new site', async () => {
+			await difmFlow.clickButton( 'start a new site' );
+			await difmFlow.validateOptionsPage();
+		} );
+
+		it( 'Enter relevant information', async () => {
+			await difmFlow.enterOptions( 'Test Site', 'Just a little test' );
+			await difmFlow.clickButton( 'Continue' );
+			await difmFlow.validateSocialPage();
+		} );
+
+		it( 'Fill out the social page', async () => {
+			await difmFlow.enterSocial( 'https://twitter.com/automattic' );
+			await difmFlow.clickButton( 'Continue' );
+			await difmFlow.validateDesignPage();
+		} );
+
+		it( 'Choose one of the free designs', async () => {
+			await difmFlow.chooseFreeDesign();
+			await difmFlow.validatePagePickerPage();
+		} );
+
+		it( 'Proceed to Checkout', async () => {
+			await difmFlow.selectPage( 'Blog' );
+			await difmFlow.selectPage( 'Services' );
```

Comment:
I would pull these two steps out into their own case so any failures would be more quickly understandable. 👍

Reply:
Good idea 👍🏻